### PR TITLE
elasticmq-server-bin: init at 0.14.6

### PR DESCRIPTION
**Pull request by peterromfeldhk**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [x] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Assured whether relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/servers/elasticmq-server-bin/default.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{ stdenv, fetchurl, jdk, jre, makeWrapper }:
+
+stdenv.mkDerivation rec {
+  name = "elasticmq-server-${version}";
```
Comment (Lassulus) [inline pkgs/servers/elasticmq-server-bin/default.nix]:
```suggestion
  pname = "elasticmq-server";
```
 the default value of name is `name = "${pname}-${version}"

File: pkgs/servers/elasticmq-server-bin/default.nix [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+    cp $src $out/share/elasticmq-server/elasticmq-server.jar
+
+    # TODO: how to add extraArgs? currently workaround is to use JAVA_TOOL_OPTIONS environment to specify properties
+    makeWrapper ${jre}/bin/java $out/bin/elasticmq-server \
+      --add-flags "-jar $out/share/elasticmq-server/elasticmq-server.jar"
+  '';
+
+  meta = with stdenv.lib; {
+    homepage = https://github.com/softwaremill/elasticmq;
```
Comment (Lassulus) [inline pkgs/servers/elasticmq-server-bin/default.nix]:
```suggestion
    homepage = "https://github.com/softwaremill/elasticmq";
```

Comment (peterromfeldhk) [reply pkgs/servers/elasticmq-server-bin/default.nix]: hmm others told me before to remove quotes from `homepage` :/

but well

Comment (Lassulus) [reply pkgs/servers/elasticmq-server-bin/default.nix]: oh yeah, there is a little movement around this, sorry: https://github.com/NixOS/rfcs/pull/45

**Comment by Lassulus** [comment]
tested with nix-preview, seems to be running.

**Pull request merged**